data/sd/legislature/Linda-K-Duba-dda1d676-f5ad-4491-b66a-cd9c8b94ffa7.yml

```diff
@@ -1,28 +0,0 @@
-id: ocd-person/dda1d676-f5ad-4491-b66a-cd9c8b94ffa7
-name: Linda K. Duba
-given_name: Linda
-family_name: Duba
-email: Linda.Duba@sdlegislature.gov
-image: https://lawmakerdocuments.blob.core.usgovcloudapi.net/photos/duba_2019.jpg
-party:
-- name: Democratic
-roles:
-- type: lower
-  jurisdiction: ocd-jurisdiction/country:us/state:sd/government
```

Comment:
@sroomf i think this file should be un-deleted? Because it is Linda's original file.